<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchSettingWidget</class>
 <widget class="QWidget" name="SearchSettingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
   <item>
    <widget class="QGroupBox" name="switch_groupbox">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="title">
      <string>功能开关</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
      <item>
       <widget class="QRadioButton" name="on_radio">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>启用壁纸插件</string>
        </property>
        <property name="text">
         <string>开启</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="off_radio">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>关闭壁纸插件</string>
        </property>
        <property name="text">
         <string>关闭</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="title">
      <string>自定义搜索过滤</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,10,0">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>程序：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="program_label">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="program_button">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,10,0">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>文档：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="document_label">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="document_button">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,10,0">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>视频：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="video_label">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>14</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="video_button">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,10,0">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>音频：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="sound_label">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>14</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="sound_button">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,10,0">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>图片：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="image_label">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>14</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="image_button">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
